### PR TITLE
Fix #378: Prepare PostgreSQL schema

### LINKED ISSUE #378: Prepare PostgreSQL schema

### PR DESCRIPTION
**Pull request by petrdvorak**

### CODE REVIEW

File: docs/sql/postgresql/create_push_server_schema.sql [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	ios_key_id VARCHAR(255),
+	ios_private_key VARCHAR(255),
+	ios_team_id VARCHAR(255),
+	ios_bundle VARCHAR(255),
+	android_private_key BYTEA,
+	android_project_id VARCHAR(255)
+);
+
+
+-- Create table for registerd devices
```
Comment (romanstrobl) [inline docs/sql/postgresql/create_push_server_schema.sql:28]: Typo: registerd

**Review by romanstrobl — APPROVED** [review]
Looks fine when compared to other DDLs, but I think we are missing foreign keys in the schema.

File: docs/sql/postgresql/create_push_server_schema.sql (hunk 1 above)
Comment (petrdvorak) [reply docs/sql/postgresql/create_push_server_schema.sql:28]: Fixed.

**Comment by petrdvorak** [comment]
I will open a new issue on the indexes.

**Pull request merged**